src/snyk/snykCode/codeSettings.ts

```diff
@@ -38,27 +38,37 @@ export class CodeSettings implements ICodeSettings {
   ) {}
 
   async checkCodeEnabled(): Promise<boolean> {
-    const settings = await this.getSastSettings();
-    if (!settings) {
-      return false;
+    let codeEnabled = false;
+    let localCodeEngineEnabled = false;
+    try {
+      const settings = await this.getSastSettings();
+      if (!settings) {
+        return false;
+      }
+      codeEnabled = settings.sastEnabled && !settings.localCodeEngine.enabled;
+      localCodeEngineEnabled = settings.localCodeEngine.enabled;
+    } catch (e) {
+      // Ignore potential command not found error during LS startup and poll
+      codeEnabled = await this.enable(false);
```

Comment:
Why do we try to enable Code here? If it's unavailable to the user, there will be 65+ seconds of polling before we get `false`